corehq/apps/export/models/new.py

```diff
@@ -792,6 +800,76 @@ def _new_from_schema(cls, schema):
         )
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = SMS_EXPORT
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
+                TableConfiguration(
+                    label="Messages",
+                    columns=[
+                        RowNumberColumn(),
+                        ExportColumn(
+                            label="Contact ID",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Timestamp",
+                            item=ExportItem(
+                                path=[PathNode(name='date')]
+                            ),
+                            selected=True,
+                        ),
+                        ExportColumn(
+                            label="Username",
+                            item=ExportItem(
+                                path=[PathNode(name='couch_recipient')],
+                                transform=USERNAME_TRANSFORM,
```

Comment:
does this transform handle cases too? this id could point to a case, mobile worker, web user, or be None. couch_recipient_doc_type gives the doc type that this id points to